controls/radheatmap/row-column-headers.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Row/Column Headers
+page_title: Row/Column Headers
+description: Row/Column Headers
+slug: radheatmap-row-column-headers
+tags: tooltip, heatmap, row, columnheader, celltooltip, heatmaprowheadersettings, heatmapcolumnheadersettings
+published: True
+position: 5
+---
+
+#  Row/Column Headers
+
+The Row/Column headers of the RadHeatMap can be further customized through  the __RowHeaderSettings__ and __ColumnHeaderSettings__ properties. 
+
+## Row Headers
+
+The customization of the row headers is possible through  the __RowHeaderSettings__ property of the RadHeatMap. This property is of type __HeatMapRowHeaderSettings__ object. Except for exposing the common properties for styling the labels, the __HeatMapRowHeaderSettings__ expose an additional one to change the position of the label to the right side of the RadHeatMap. 
+
+ * __LabelsPanelPosition__: A property of type __HeatMapRowLabelsPanelPosition__ that gets or sets the row headers position. This is an enumeration and it allows the following values:
+	* __Left__: The row labels will be position on the left side of the RadHeatMap. (Default value)
+	* __Right__: The row labels will be position on the right side of the RadHeatMap.
+
+#### __[XAML] Example 1: Modifying row headers__
+
+{{region radheatmap-row-column-headers_0}}
+	<telerik:RadHeatMap >            
+		<telerik:RadHeatMap.RowHeaderSettings>
+			<telerik:HeatMapRowHeaderSettings LabelsPanelPosition="Right"/>
+		</telerik:RadHeatMap.RowHeaderSettings>
+	</telerik:RadHeatMap>
+{{endregion}}
+
+## Column Headers
+
+The customization of the column headers is possible through the __ColumnHeaderSettings__ property of the RadHeatMap. This property is of type __HeatMapColumnHeaderSettings__ object. Except for exposing the common properties for styling the labels, the __HeatMapColumnHeaderSettings__ expose two additional properties to change the position of the column headers and to rotate the labels.
+
+ * __LabelsPanelPosition__: A property of type __HeatMapColumnLabelsPanelPosition__ that gets or sets the column headers position. This is an enumeration and it allows the following values:
+	* __Top__: The column headers will be position at the top of the RadHeatMap. (Default value)
+	* __Bottom__: The column headers will be position at the bottom of the RadHeatMap.
+	
+ * __LabelRotationAngle__: A property of type double that gets or sets the rotation angle applied to the label. 
```

Comment:
Is the LabelRotationAngle property available only for the ColumnHeaderSettings?

Reply:
Yes, this property is available only for the column header settings.